e2e_tests/test_suites/ospolicies/ospolicies.go

```diff
@@ -240,17 +240,19 @@ func testCase(ctx context.Context, testSetup *osPolicyTestSetup, tests chan *jun
 	} else {
 		logger.Printf("Running TestCase %q", tc.Name)
 		runTest(ctx, tc, testSetup, logger)
-		if tc.Failure != nil {
-			rerunTC := junitxml.NewTestCase(testSuiteName, strings.TrimPrefix(tc.Name, fmt.Sprintf("[%s] ", testSuiteName)))
-			wg.Add(1)
-			go func() {
-				defer wg.Done()
-				logger.Printf("Rerunning TestCase %q", rerunTC.Name)
-				runTest(ctx, rerunTC, testSetup, logger)
-				rerunTC.Finish(tests)
-				logger.Printf("TestCase %q finished in %fs", rerunTC.Name, rerunTC.Time)
-			}()
-		}
+		/*
+			if tc.Failure != nil {
+				rerunTC := junitxml.NewTestCase(testSuiteName, strings.TrimPrefix(tc.Name, fmt.Sprintf("[%s] ", testSuiteName)))
+				wg.Add(1)
+				go func() {
+					defer wg.Done()
+					logger.Printf("Rerunning TestCase %q", rerunTC.Name)
+					runTest(ctx, rerunTC, testSetup, logger)
+					rerunTC.Finish(tests)
+					logger.Printf("TestCase %q finished in %fs", rerunTC.Name, rerunTC.Time)
+				}()
+			}
```

Comment:
nit: do you want to just remove it?

Reply:
Not right now, I think I may want add this back in once I get tests working, I'll probably just want to tweak the test timeouts.